checker/src/org/checkerframework/checker/regex/RegexVisitor.java

```diff
@@ -91,9 +91,17 @@ public Void visitMethodInvocation(MethodInvocationTree node, Void p) {
                 LiteralTree literal = (LiteralTree) group;
                 int paramGroups = (Integer) literal.getValue();
                 ExpressionTree receiver = TreeUtils.getReceiverTree(node);
+                if (receiver == null) {
+                    // When checking java.util.regex.Matcher, calls to group (and other methods) don't
```

Comment:
This comment is not clear to me.
Do you mean checking the file `java/util/regex/Matcher.java`?